test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -1,27 +1,48 @@
 #!/usr/bin/env bash
 
 function usage() {
-    echo "Usage: push_antrea.sh [--prometheus] [-h|--help]"
+    echo "Usage: push_antrea.sh [--prometheus] [-fa|--flow-agg] [-h|--help]
+    Push the latest Antrea image to all vagrant nodes and restart the Antrea daemons
+          --prometheus           Deploy Prometheus service to scrape metrics from Antrea Agents and Controllers
+          --flow-agg (collector) Deploy Flow Aggregator service to get all flows from Flow Exporter at Antrea Agents
```

Comment:
The value of argument `--flow-agg` is the external IPFIX collector IP right? I'm worried about misunderstanding here, maybe two separate argument `--flow-agg` to enable flow aggregator and `--collector-addr` to provide the external IPFIX collector IP?

Reply:
I agree with @dreamtalen, I find it confusing.